### PR TITLE
Add Playground Tools issue templates

### PR DESCRIPTION
**Pull request by flexseth**
## What?
Adds issue template starting point for Playground Tools

## Why?
Creates guidelines for better error reporting

## How?
Creating an issue template

## Testing Instructions
1. [Allow issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#configuring-the-template-chooser) on the 
`Playground Tools` repo
2. Check or merge PR
3. Open a new issue and use the `Playground Tools issues - wp-now, VS Code extension, utilities` template
4. See if it suits the needs or needs updating

### CODE REVIEW

**Review by flexseth — COMMENTED** [review]
@adamziel - if you can please review this PR to add an issues template for error reporting, it may be improved when more of the known issues docs come online.

For now, it can set the stage for some info we'd like to collect when reporting errors. 

It's a little less detailed than the Playground online issue template, because the end users is assumed to be a little more advanced.

File: playground-tools-issues---wp-now--vs-code-extension--utilities.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ assignees: ''
 
 ## First make sure that you search for related issues in the `Playground Tools` repo's [issues](https://github.com/WordPress/playground-tools/issues)
 
-if not found...
+if not found... use whatever sections you want below to describe your issue.
```
Comment (flexseth) [inline playground-tools-issues---wp-now--vs-code-extension--utilities.md]: TLDR; you don't have to provide all of the info

Does this help or make the issue template still too complicated, @adamziel?

**Review by flexseth — COMMENTED** [review]
Please take a look at this and let me know what you think, @adamziel

File: playground-tools-issues---wp-now--vs-code-extension--utilities.md (hunk 1 above)
Comment (adamziel) [reply playground-tools-issues---wp-now--vs-code-extension--utilities.md]: @flexseth it's just it's a wall of text to process. Let's make it a simple checklist and link out to a doc page if needed.

Comment (flexseth) [reply playground-tools-issues---wp-now--vs-code-extension--utilities.md]: > @flexseth it's just it's a wall of text to process. Let's make it a simple checklist and link out to a doc page if needed.

Alright, reworking similarly to the other issue template!

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+labels: bug
+assignees: ''
+
+---
+
+<!-- 
+Report issues with online Playground instances [here.](https://github.com/WordPress/wordpress-playground/issues)
+-->
+
+## How are you using the Playground?
```
Comment (adamziel) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
## Which Playground tool did you experience an issue with? (Wp-now, VS Code extension, etc.)

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+<!--
```
Comment (adamziel) [inline .github/ISSUE_TEMPLATE.md]: And then we could remove this comment or make it a single line of text to save verticsl space

**Review by flexseth — COMMENTED** [review]
changes made per request

**Comment by ironnysh** [comment]
Suggestion: add a GitHub workflow to auto-label issues when submitting (see [example on WordPress Learn](https://github.com/WordPress/Learn/blob/trunk/.github/workflows/label-feedback-from-comment.yml))

**Comment by adamziel** [comment]
Shouldn't this be a `.yml` file living in `.github/ISSUE_TEMPLATE`? Or will it work as a .md file living in .github?

**Comment by flexseth** [comment]
It depends how it was created, I'll have to go back through the workflow to figure out how it should be named

**Comment by flexseth** [comment]
> Shouldn't this be a `.yml` file living in `.github/ISSUE_TEMPLATE`? Or will it work as a .md file living in .github?

TLDR; It'll work as submitted :)

---

The way I created it (screen flow attached) - used the [Markdown issue syntax.](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#markdown-issue-template)

https://github.com/WordPress/playground-tools/assets/3792502/06aba673-27ac-431e-8b50-8361ae961a28


---
Looks like it can be either. 

## Advanced issue templates
For more advanced issue flows, utilizing `GitHub Actions` etc - you can provide additional metadata via `YAML`. This can help with automatically labeling or assigning issues, and allows additional markup options (more similar to HTML input)

[`YAML` syntax for issue templates](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms#about-yaml-syntax-for-issue-forms)


## Advancing the PR
Want to start with Markdown and see if the `YAML` implementation provides more value, via automation, in the future?

**Comment by flexseth** [comment]
> Suggestion: add a GitHub workflow to auto-label issues when submitting (see [example on WordPress Learn](https://github.com/WordPress/Learn/blob/trunk/.github/workflows/label-feedback-from-comment.yml))

Awaiting feedback about implementing automated workflows, or starting out with a basic issue template

Here's the PR though!

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,32 @@
+---
+name: Playground Tools issues - wp-now, VS Code extension, utilities
+about: Did a tool not work the way you want it to? Let us know!
```
Comment (adamziel) [inline .github/ISSUE_TEMPLATE.md]:
```suggestion
name: Bug report
about: Report a bug with Playground tools, e.g. wp-now, VS Code extension, Playground block, Playground plugin
```

Comment (adamziel) [reply .github/ISSUE_TEMPLATE.md]: Let's adjust this and it should be good to go. Thank you for your work!

**Review by flexseth — COMMENTED** [review]
@adamziel - this should be it!!

**Comment by flexseth** [comment]
> Shouldn't this be a `.yml` file living in `.github/ISSUE_TEMPLATE`? Or will it work as a .md file living in .github?

It's throwing a warning on the Playground Tools (this) [issue template.](https://github.com/flexseth/playground-tools/blob/issue-templates/playground-tools/.github/ISSUE_TEMPLATE.md)

---

## Screenshot

image

---

Strangely, I don't see the same notice on the [primary Playground repo issues template](https://github.com/flexseth/wordpress-playground/blob/issue-templates/playground-online/.github/playground-wordpress-net---issues.md) 

Want me to figure it out?
